site/includes/sidebar.pug

```diff
@@ -1,11 +1,12 @@
-.spectrum-Site-sideBar#site-sidebar
-  a.spectrum-Site-sideBarHeader.js-fastLoad(href='index.html')
-    svg.spectrum-Site-logo.spectrum-Icon.spectrum-Icon--sizeL(viewBox="0 0 30 26" focusable="false" aria-hidden="true" aria-label="Adobe logo")
-      polygon(points="19,0 30,0 30,26")
-      polygon(points="11.1,0 0,0 0,26")
-      polygon(points="15,9.6 22.1,26 17.5,26 15.4,20.8 10.2,20.8")
-    h2.spectrum-Site-title.spectrum-Heading.spectrum-Heading--M Spectrum CSS
+unless process.env.VRT
+  .spectrum-Site-sideBar#site-sidebar
+    a.spectrum-Site-sideBarHeader.js-fastLoad(href='index.html')
+      svg.spectrum-Site-logo.spectrum-Icon.spectrum-Icon--sizeL(viewBox="0 0 30 26" focusable="false" aria-hidden="true" aria-label="Adobe logo")
```

Comment:
hmm, usually not big fan of adding directly the SVG into pug.
If you place the SVG image in an image-dir you could include it like
```pug

  svg(width="30px", height="26px")
    use(xlink:href="./images/svg/spectrum-size-logo.svg#spectrum-logo")

Reply:
Looks like the inline SVG is there for some time. The change I made here is conditional render the side bar depends on the `VRT` env variable.

Reply:
maybe we keep this in mind. Not really part of this task.